app/models/org.rb

```diff
@@ -185,6 +198,10 @@ def self.human_attribute_name(attr, options = {})
     HUMANIZED_ATTRIBUTES[attr.to_sym] || super
   end
 
+  # ===========================
+  # = Public instance methods =
+  # ===========================
+
   def links
     super() || { "org": [] }
   end
```

Comment:
Just noticed this, should we refactor to a default and remove this method?

Reply:
We would also need a rake:task to fixup any existing data

Reply:
yes and yes :)